pkg/routes/v1.go

```diff
@@ -565,4 +565,10 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
 	{
 		pkpassGroup.GET("", h.PkPass.GeneratePkPass)
 	}
+
+	emojisGroup := v1.Group("emojis")
+	{
+		emojisGroup.POST("list", h.Emojis.ListEmojis)
```

Comment:
get all just need "/api/v1/emojis"
get list just need something like "/api/v1/emojis?query=ftm,eth"
I recommend create just 1 api and if no query param then return all, if yes then add query